recipes-containers/podman/seccomp.json

```diff
@@ -0,0 +1,833 @@
+{
```

Comment:
Where does this file come from? I see that one should use https://github.com/containers/common?tab=readme-ov-file#seccomp to generate the profile. Also, there is [this](https://github.com/containers/common/blob/6260b5fe114cbe15399268eeb58cc7083ac5af36/pkg/seccomp/seccomp.json#L4) pre-generated file but I'm not sure if it is good enough for your use-case

Reply:
@alexhulbert

Reply:
https://github.com/containers/common/issues/1988#issuecomment-2352869014